app/javascript/components/ReservseForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { createReservations } from "../redux/reservations/reservationsSlice";
+import { fetchDoctors } from "../redux/doctors/doctorsSlice";
+
+const ReserveForm = () => {
```

Comment:
- [ ] When I add a doctor, and they try to make a reservation with that doctor, it does not work on my end:

![image](https://github.com/Benawi/Doctor_Appointment-Booking_App/assets/57421823/de1a9076-51d3-4a2c-9192-3572275b052e)

![image](https://github.com/Benawi/Doctor_Appointment-Booking_App/assets/57421823/a57fd185-5aa5-4a07-bdc5-4f941b30df06)

This seems to happen because the doctor ID that is being sent to the server is "0".

Kindly, correct this behaviour so your app works as intended. 👍